src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -40,7 +40,7 @@ def validate_site_create(cmd, namespace):
             }
         }
         validation = client.validate(resource_group_name, validation_payload)
-        if validation.status.lower() == "failure":
+        if validation.status.lower() == "failure" and validation.error.code != 'SiteAlreadyExists':
```

Comment:
Do we need case insensitive check here just to be sure?

Reply:
How does this help - when someone actually trying to create a new ASP or APP with the same name accidentally? If it doesn't then we should just remove the validation check in these commands.

Reply:
The code will just proceed to create the ASP/WebApp on name conflicts, which is the old behavior. I am keeping the validation logic here as I assume it does other kinds of validations which are still beneficial? However if the naming conflicts is the only thing the validate API checks, then I agree we should remove the validation check completely.

Reply:
The API does do more than just name validation - keep it for now, & i will update az webapp up as needed.

Reply:
I am merging. For error code change, case sensitive check is fine, as the code is supposed to be treated more prudently and less randomness on case change